tst/teststandard/grpauto.tst

```diff
@@ -11,6 +11,15 @@
 
 gap> START_TEST("grpauto.tst");
 
```

Comment:
For the record, the version with all packages took 1330s and did not run out of memory (`-o 1g -K 2g`).